python/ray/tune/execution/ray_trial_executor.py

```diff
@@ -388,14 +377,15 @@ def _maybe_use_cached_actor(self, trial, logger_creator) -> Optional[ActorHandle
         return actor
 
     def _setup_remote_runner(self, trial):
-        trial.init_logdir()
         # We checkpoint metadata here to try mitigating logdir duplication
         self._trials_to_cache.add(trial)
-        logger_creator = partial(
-            _noop_logger_creator,
-            logdir=trial.logdir,
+
+        trainable_kwargs = _get_trainable_kwargs(
+            trial,
+            additional_kwargs=self._trainable_kwargs,
             should_chdir=self._chdir_to_trial_dir,
         )
+        logger_creator = trainable_kwargs["logger_creator"]
```

Comment:
Created an issue to deprecate the need for `additional_kwargs`: https://github.com/ray-project/ray/issues/33265